analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/Checks/LocksReleasedAllPathsBase.cs

```diff
@@ -55,6 +55,21 @@ public override ProgramState PostProcess(SymbolicContext context)
                 {
                     return ProcessMonitorExit(context, invocation);
                 }
+                else if (invocation.TargetMethod.Is(KnownType.System_Threading_ReaderWriterLockSlim, "EnterReadLock")
+                         || invocation.TargetMethod.Is(KnownType.System_Threading_ReaderWriterLockSlim, "TryEnterReadLock")
```

Comment:
It took me a while to compare these with docs. I'd prefer to have them alphabetically, but it will not matter at all after moving them to the `IsAny`.